### PR TITLE
Bug 1886973: Hide the LocalVolumeDiscoveryResult CRD from the UI

### PR DESCRIPTION
**Pull request by huffmanca**
Adds the LocalVolumeDiscoveryResult CRD to the list of internal objects, preventing it from being displayed in the UI.

### CODE REVIEW

File: manifests/4.7/local-storage-operator.v4.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -102,6 +102,7 @@ metadata:
       Configure and use local storage volumes in kubernetes and OpenShift.
       OpenShift 4.2 and above is only supported OpenShift versions.
     olm.skipRange: ">=4.3.0 <4.7.0"
+    operators.operatorframework.io/internal-objects: '["localvolumediscoveryresults.local.storage.openshift.io"]'
```
Comment (gnufied) [inline manifests/4.7/local-storage-operator.v4.7.0.clusterserviceversion.yaml:105]: for completeness sake, can you also update opm-bundle directory's CSV?

Comment (huffmanca) [reply manifests/4.7/local-storage-operator.v4.7.0.clusterserviceversion.yaml:105]: This has been included.

File: opm-bundle/manifests/local-storage-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -305,7 +306,7 @@ spec:
                 serviceAccountName: local-storage-operator
                 containers:
                   - name: local-storage-operator
-                    image: quay.io/openshift/origin-local-storage-operator:latest
+                    image: quay.io/huffmanca/lso:latest
                     ports:
```
Comment (gnufied) [inline opm-bundle/manifests/local-storage-operator.clusterserviceversion.yaml:310]: we should leave this image names alone though..

Comment (huffmanca) [reply opm-bundle/manifests/local-storage-operator.clusterserviceversion.yaml]: Apologies. These were leftover from testing, but should be resolved.

**Pull request merged**